# HG changeset patch
# User Nicholas Musolino <musolino@mit.edu>
# Date 1342473410 14400
#      Mon Jul 16 17:16:50 2012 -0400
# Branch stable
# Node ID 70f86a64c412bc914b3fa641e935eacc11cf62a8
# Parent  e2ac6e2c588a3c33a3a53eea2f045d8b6cacd4c1
* voronoi.m: Fix cut and paste error.

diff --git a/scripts/geometry/voronoi.m b/scripts/geometry/voronoi.m
--- a/scripts/geometry/voronoi.m
+++ b/scripts/geometry/voronoi.m
@@ -121,18 +121,18 @@ function [vx, vy] = voronoi (varargin)
   ymax = max (y(:));
   ymin = min (y(:));
   xdelta = xmax - xmin;
   ydelta = ymax - ymin;
   scale = 2;
 
   xbox = [xmin - scale * xdelta; xmin - scale * xdelta; ...
           xmax + scale * xdelta; xmax + scale * xdelta];
-  ybox = [xmin - scale * xdelta; xmax + scale * xdelta; ...
-          xmax + scale * xdelta; xmin - scale * xdelta];
+  ybox = [ymin - scale * ydelta; ymax + scale * ydelta; ...
+          ymax + scale * ydelta; ymin - scale * ydelta];
 
   [p, c, infi] = __voronoi__ ("voronoi",
                               [[x(:) ; xbox(:)], [y(:); ybox(:)]],
                               opts{:});
 
   idx = find (! infi);
   ll = length (idx);
   c = c(idx).';
